docs/BUILD.md

```diff
@@ -119,29 +119,29 @@ Clone the `openvino_tensorflow` repository:
   ```
 
 ###  2.1. <a name='BuildInstructions'></a>Build Instructions
-Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enable only the Python APIs, TensorFlow C++ libraries built from source is required to use the C++ APIs.
+Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enables only the Python APIs. TensorFlow C++ libraries built from source is required to use the C++ APIs.
 
-1. Pulls compatible prebuilt TF package from PyPi, clones and builds OpenVINO™ from source. The arguments are optional, if not provided then default versions as specified in build_ovtf.py will be used. 
+1. Pulls compatible prebuilt TensorFlow package from PyPi, clones and builds OpenVINO™ from source. The arguments are optional. If any argument is not provided, then the default versions as specified in build_ovtf.py will be used. 
 
         python3 build_ovtf.py --tf_version=v2.5.1 --openvino_version=2021.4.1
 
-2. Pulls compatible prebuilt TF package from PyPi. Uses OpenVINO™ binary from specified location.
+2. Pulls compatible prebuilt TensorFlow package from PyPi. Uses OpenVINO™ binary from specified location.
 
         python3 build_ovtf.py --use_openvino_from_location=/opt/intel/openvino_2021.4.689/ --cxx11_abi_version=1
 
-3. Uses pre-built TF from the given location ([refer the TensorFlow build instructions](#tensorflow)). Pulls and builds OpenVINO™ from source. Use this if you need to build **OpenVINO™ integration with TensorFlow** frequently without building TF from source everytime.
+3. Uses pre-built TensorFlow from the given location ([refer the TensorFlow build instructions](#tensorflow)). Pulls and builds OpenVINO™ from source. Use this if you need to build **OpenVINO™ integration with TensorFlow** frequently without building TensorFlow from source everytime.
```

Comment:
Did we ever mention about build_tf.py? The specified TensorFlow location is expected to be populated by build_tf.py.

Reply:
Yes, we refer to "build_tf.py" in the same file, and also the hyperlink "refer the TensorFlow build instructions" will take user to "TensorFlow" build section which has instructions to use "build_tf.py"